src/vendor/components/navigation/slider.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
```

Comment:
Rename the file so that it matches the casing of the class: `src/vendor/components/navigation/Slider.jsx`.